manim/mobject/mobject.py

```diff
@@ -2641,26 +2690,30 @@ def init_alignments(alignments, num, mapping, name, dir_):
             raise ValueError(
                 'flow_order must be one of the following values: "dr", "rd", "ld" "dl", "ru", "ur", "lu", "ul".',
             )
-        flow_order = mapper[flow_order]
+        flow_order_method = mapper[flow_order]
```

Comment:
As a minor nitpick, the lambdas are functions, not methods:
```suggestion
        flow_order_function = mapper[flow_order]
```